paypal/paypal.lua

```diff
@@ -60,7 +60,7 @@ local clientSecret = inputVar.client_secret
 
 r, c, h = https.request{
     method = 'POST',
-    url = 'https://api.sandbox.paypal.com/v1/oauth2/token',
+    url = 'https://api.sandbox.paypal.com',
```

Comment:
Based on the code I suppose this won't work anymore? you still left hardcoded domain for authorization retrieval, but shouldn't you request /v1/oauth2/token to get the token? I suppose you have to use inputVar.paypal_domain and /v1/oauth2/token to get this working.